hono/src/services/swagger.ts

```diff
@@ -0,0 +1,28 @@
+import { SwaggerUI } from '@hono/swagger-ui'
+import { Hono } from 'hono'
+import swaggerJson from '../../docs/swagger.json'
+import type { Variables } from '../utils/inject-db'
+
+const app = new Hono<{ Variables: Variables }>()
+
+app.get('/doc', (c) => {
+  return c.json(swaggerJson)
+})
+
+app.get('/', (c) => {
+  return c.html(`
+    <html lang="en">
+      <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="description" content="Custom Swagger" />
```

Comment:
I'd update the description to "A wardrobe logging, composition, and organization app"